docs/sdk/index.mdx

```diff
@@ -87,3 +91,111 @@ Supported SDKs for server-side:
   </div>
 
 </div>
+
+
+## Features available
+
+The table below illustrates all features available in each one of the available SKDs from Bucketeer.
+
+<div className="center-table">
+  <table>
+    <thead>
+      <tr>
+        <th>Feature</th>
+        <th><a href="/sdk/client-side/android">Android</a> </th>
+        <th><a href="/sdk/client-side/ios">iOS</a></th>
+        <th><a href="/sdk/client-side/flutter">Flutter</a></th>
+        <th><a href="/sdk/client-side/javascript">Javascript</a></th>
+        <th><a href="/sdk/server-side/go">Go</a></th>
+        <th><a href="/sdk/server-side/node-js">Node Js</a></th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Evaluating user</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Variation types</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Updating user evaluations</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Updating user evaluations in real-time</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><UncheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Reporting custom events</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Flushing events</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>User attributes configuration</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Updating user attributes</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Getting user information</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Getting evaluation details</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Listening to evaluation updates</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Background mode</td>
+        <td><UncheckSVG/></td>
```

Comment:
Android has a background mode, too.
The difference between it and iOS is that iOS requires manual configuration to use it. Android will automatically do the background tasks when available.